package.json

```diff
@@ -17,9 +17,13 @@
   },
   "dependencies": {
     "code-push": "1.5.2-beta",
-    "semver": "^5.1.0"
+    "semver": "^5.1.0",
+    "rnpm": "1.4.1"
   },
-  "devDependencies": {
-    "react-native": "0.19.0"
+  "peerDependencies": {
+    "react-native": "^0.19.0"
+  },
+  "scripts": { 
+    "postinstall": "cd ../../ && rnpm link react-native-code-push"
```

Comment:
I noticed that your RNPM PR just got merged, but I'm not sure if that would address the scenario I mentioned above. If a dev manually linked another RN module by appending the project name to an existing include statement in their build.gradle file, when they installed CodePush, it would then break their build file.